gapii/cc/spy.cpp

```diff
@@ -137,7 +137,12 @@ Spy::Spy()
   // Use a "localabstract" pipe on Android to prevent depending on the traced
   // application having the INTERNET permission set, required for opening and
   // listening on a TCP socket.
-  mConnection = ConnectionStream::listenPipe("gapii", true);
+  std::string pipe = "gapii";
+  char* envPipe = getenv("GAPII_PIPE_NAME");
```

Comment:
I wish there were a nicer way to pass things in. But I haven't figured one out yet either. We use this for the port file as well.